tests/unit/querybuf.tcl

```diff
@@ -63,4 +63,22 @@ start_server {tags {"querybuf slow"}} {
         assert {[client_query_buffer test_client] > 0 && [client_query_buffer test_client] < $orig_test_client_qbuf}
         $rd close
     }
-}
+
+    test "query buffer shouldn't be resized with fat argv" {
+        set rd [redis_client]
+        $rd client setname test_client
+        $rd write "*3\r\n\$3\r\nset\r\n\$1\r\na\r\n\$1000000\r\n"
+        $rd flush
+        set retry 10
+        while {[incr retry -1]} {
+            after 100
```

Comment:
considering there are less than 10 clients, we can actually be certain this client will be handled in the first cron trigger, so spending a full second in this test is excessive, right?
so we can actually reduce `retry` to just 2 and it should be enough,
maybe we can set `hz` to 100, and reduce the `after` to 10?
or actually why do we need a loop, we can just sleep 100ms and make the check with a simple assert_equals.

Reply:
actually, maybe we do wanna wait for 2 seconds and see the query buffer being shrunk for the idle connection?
i see this test group is marked with `slow`, so it could be ok for the test to take a few seconds.

Reply:
I'm leaning towards your later point, let's cover more scenarios